config/routes.rb

```diff
@@ -1,3 +1,4 @@
 Rails.application.routes.draw do
+  get 'api/open-library', to: 'open_library#index', at: "open_library_api_get"
```

Comment:
https://guides.rubyonrails.org/routing.html important read this

please find info about the REST API, the index method doesn’t use in this case

```suggestion
   namespace :api
     resources :open_library, only: :show
   end
```